<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gcomps</class>
 <widget class="QDialog" name="gcomps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Группы ПК</string>
  </property>
  <widget class="QTableView" name="t_comps">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>5</y>
     <width>336</width>
     <height>321</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerItem</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>24</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>22</number>
   </attribute>
  </widget>
  <widget class="QListView" name="l_gcomps">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>146</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>167</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="b_add_gr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/images/buttons/add.png</normaloff>:/images/buttons/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b_edit_gr">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>330</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/images/buttons/edit.png</normaloff>:/images/buttons/edit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="b_del_gr">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/images/buttons/remove.png</normaloff>:/images/buttons/remove.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_gcomps">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>176</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>335</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Перенести в:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_setg_ok">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>330</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/images/buttons/apply.png</normaloff>:/images/buttons/apply.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="b_setg_cancel">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>330</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/images/buttons/cancel.png</normaloff>:/images/buttons/cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>l_gcomps</tabstop>
  <tabstop>t_comps</tabstop>
  <tabstop>b_add_gr</tabstop>
  <tabstop>b_edit_gr</tabstop>
  <tabstop>b_del_gr</tabstop>
  <tabstop>cb_gcomps</tabstop>
  <tabstop>b_setg_ok</tabstop>
  <tabstop>b_setg_cancel</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
